<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML><!-- #BeginTemplate "/Templates/dreamtemplate18.dwt" --><!-- DW6 -->
<HEAD>
<!-- #BeginEditable "doctitle" --> 
<TITLE></TITLE>
<!-- #EndEditable --> 
<META http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<LINK rel="stylesheet" href="general.css" type="text/css">
<!-- #BeginEditable "metaTags" --> 
<META name="keywords" content="">
<META name="description" content="">
<style type="text/css">
<!--
.STYLE5 {
	font-size: 16px;
	font-weight: bold;
}
.STYLE6 {font-size: 14px}
.STYLE7 {font-size: 14}
.STYLE11 {color: #800040}
.STYLE12 {color: #800000}
.STYLE13 {
	font-size: 18px;
	font-weight: bold;
}
.STYLE22 {color: #990033}
.STYLE23 {color: #990066}
.STYLE25 {font-size: 18px; font-weight: bold; color: #990000; }
-->
</style>
<!-- #EndEditable --> 
<style type="text/css">
<!--
.STYLE4 {
	font-size: medium;
	color: #000066;
	font-weight: bold;
}
-->
</style>
</HEAD>
<BODY>
<TABLE width="90%" border="0" align="center" cellpadding="0" cellspacing="0" id="maintable">
  <TR> 
    <TD><IMG src="images/topleft.gif" alt="" width="10" height="72"></TD>
    <TD width="100%" valign="bottom"> 
      <TABLE width="100%" border="0" cellspacing="0" cellpadding="0">
        <TR> 
          <TD bgcolor="#FFFFFF"><div align="right"><img src="figure/ShanghaiTech_logo.jpg" width="150" height="150"></div></TD>
        </TR>
        <TR> 
          <TD> 
            <TABLE border="0" cellspacing="0" cellpadding="0">
              <TR> 
                <TD><IMG src="images/buttonleft.gif" alt="" width="9" height="30"></TD>
                <TD background="images/buttonmiddle.gif" class="tabs" width="100" valign="middle"><A href="index.htm" class="tabs">Home</A>                </TD>
                <TD valign="middle"><IMG src="images/buttonright.gif" alt="" width="10" height="30"></TD>
                <TD valign="middle"><IMG src="images/buttonleft.gif" alt="" width="9" height="30"></TD>
                <TD background="images/buttonmiddle.gif" class="tabs" width="100" valign="middle"><A href="people.htm" class="tabs">People</A></TD>
                <TD valign="middle"><IMG src="images/buttonright.gif" alt="" width="10" height="30"></TD>
                <TD valign="middle"><IMG src="images/buttonleft.gif" alt="" width="9" height="30"></TD>
                <TD background="images/buttonmiddle.gif" class="tabs" width="100" valign="middle"><A href="publications.htm" class="tabs">Publications</A></TD>
                <TD valign="middle"><IMG src="images/buttonright.gif" alt="" width="10" height="30"></TD>
                <TD valign="middle"><IMG src="images/buttonleft.gif" alt="" width="9" height="30"></TD>
                <TD background="images/buttonmiddle.gif" class="tabs" width="100" valign="middle"><A href="software.htm" class="tabs">Software</A></TD>
                <TD valign="middle"><IMG src="images/buttonright.gif" alt="" width="10" height="30"></TD>
                <TD valign="middle"><IMG src="images/buttonleft.gif" alt="" width="9" height="30"></TD>
                <TD background="images/buttonmiddle.gif" class="tabs" width="100" valign="middle"><A href="positions.htm" class="tabs">Positions</A></TD>
                <TD><IMG src="images/buttonright.gif" alt="" width="10" height="30"></TD>
                <TD><IMG src="images/buttonleft.gif" alt="Software_left" width="9" height="30"></TD>
                <TD background="images/buttonmiddle.gif" class="tabs" width="100" valign="middle"><A href="contact.htm" class="tabs">Contact</A></TD>
                <TD><IMG src="images/buttonright.gif" alt="software_right" width="10" height="30"></TD>
                <TD>&nbsp;</TD>
              </TR>
            </TABLE>
          </TD>
        </TR>
      </TABLE>
    </TD>
    <TD><IMG src="images/topright.gif" alt="" width="9" height="71"></TD>
  </TR>
  <TR> 
    <TD width="10"><IMG src="images/toplcorner.gif" alt="" width="10" height="25"></TD>
    <TD background="images/topmiddle.gif" width="100%">&nbsp;</TD>
    <TD width="9"><IMG src="images/toprcorner.gif" alt="" width="9" height="25"></TD>
  </TR>
  <TR> 
    <TD width="10">&nbsp;</TD>
    <TD width="100%" valign="top"> 
      <TABLE width="100%" border="0" cellspacing="5" cellpadding="0">
        <TR> 
          <TD valign="top"><!-- #BeginEditable "Text Table" -->
            <p>&nbsp;</p>
            <p><span class="STYLE25">Publications </span>(<a href="https://scholar.google.com/citations?hl=en&user=evC66XkAAAAJ&view_op=list_works">Full list in Google Scholar</a>) </p>
            <p class="STYLE13 STYLE6 STYLE22 STYLE6 STYLE6">Journals</p>
            <p><strong>2019</strong></p>
            <ul>
              <li>
                <p>Yong  Liu, Min Wu*, Chenghao Liu, Xiao-Li Li, <strong>Jie  Zheng*</strong>. SL2MF: Predicting synthetic lethality in human cancers via logistic  matrix factorization. <em>IEEE/ACM  Transactions on Computational Biology and Bioinformatics</em> (TCBB) (special  issue of GIW 2018), accepted, 2019 (<strong>IF</strong> = 2.428).</p>
              </li>
              <li>
                <p>Ket  Hing Chong, Sandhya Samarasinghe, Don Kulasiri*, <strong>Jie Zheng</strong>. Mathematical modelling of core regulatory mechanism in  p53 protein that activates apoptotic switch. <em>Journal of Theoretical Biology</em>, Vol. 462 (7 Feb.), pp. 134-147,  2019 (<strong>IF</strong> = 1.833).</p>
              </li>
            </ul>
            <p>&nbsp;</p>
            <p><strong>2018</strong></p>
            <ul>
              <li>
                <p>Lichun  Ma, <strong>Jie Zheng*</strong>. Single-cell gene  expression data analysis reveals beta-cell dysfunction and deficit mechanisms  in type 2 diabetes. <em>BMC Bioinformatics</em>,  19(Suppl 19):515, 2018 (special issue of GIW 2018) (<strong>IF</strong> = 2.213).</p>
              </li>
              <li>
                <p>Xinrui  Zhou, Rui Yin, Chee Keong Kwoh*, <strong>Jie Zheng*</strong>.  A context-free encoding scheme of protein sequences for predicting antigenicity  of diverse influenza A viruses. <em>BMC  Genomics</em>, 19(Suppl 10):936, 2018 (special issue of GIW 2018) (<strong>IF</strong> = 3.730).</p>
              </li>
              <li>
                <p>Rui  Yin, Xinrui Zhou, <strong>Jie Zheng</strong>, Chee  Keong Kwoh*. Computational identification of physiochemical signatures for host  tropism of influenza A virus. <em>Journal of  Bioinformatics and Computational Biology </em>(JBCB), Vol. 16, No. 6, 1840023, 2018  (special issue of GIW 2018) (<strong>IF</strong> = 0.991).</p>
              </li>
              <li>
                <p>Haifen Chen,  Devamuni A.K. Maduranga, Piyushkumar Mundra, <strong>Jie Zheng*</strong>. Bayesian data fusion of gene expression and histone  modification profiles for inference of gene regulatory network. <em>IEEE/ACM Transactions on Computational  Biology and Bioinformatics</em> (TCBB), Early Access, 2018 (<strong>IF</strong> = 2.428).</p>
              </li>
              <li>
                <p>Hongfei Liu,  Paolo A Lorenzini, Fan Zhang, Shaohai Xu, Mei Su M Wong, <strong>Jie Zheng</strong>, Xavier Roca*. Alternative splicing analysis in human  monocytes and macrophages reveals MBNL1 as major regulator. <em>Nucleic Acids Research</em>, 46 (12), pp.  6069 &ndash; 6086, 2018 (<strong>IF</strong> = 11.561).</p>
              </li>
              <li>
                <p>Ket Hing Chong, Xiaomeng Zhang, <strong>Jie Zheng</strong>*. Dynamical analysis of  cellular ageing by modeling of gene regulatory network based attractor  landscape. <em>PLOS ONE</em>, 13(6), e0197838,  2018 (<strong>IF</strong> = 2.766).</p>
              </li>
              <li>
                <p>Xiaomeng Zhang#, Ket Hing  Chong# (joint first authors), <strong>Jie  Zheng*</strong>. A Monte Carlo method for in silico modeling and visualization of  Waddington&rsquo;s epigenetic landscape with intermediate details. <em>bioRxiv</em> 310771, 2018. </p>
              </li>
              <li>
                <p>Xinrui Zhou, <strong>Jie Zheng</strong>, Xaverius Ivan Fransiskus, Rui Yin, Shoba Ranganathan,  Vincent T. K. Chow, Chee Keong, Kowh. Computational analysis of the receptor  binding specificity of novel influenza A/H7N9 viruses. <em>BMC Genomics</em> (special issue of APBC 2018) 19 (Suppl 2):88, 2018 (<strong>IF</strong> = 3.730).</p>
              </li>
              <li>
                <p>Bobby Ranjan, Ket Hing Chong, <strong>Jie Zheng*</strong>. Composite mathematical  modeling of calcium signaling behind neuronal cell death in Alzheimer&rsquo;s  disease. <em>BMC Systems Biology</em> (special  issue of APBC 2018) 12(Suppl 1):10, 2018 (<strong>IF</strong> = 2.050).</p>
              </li>
              <li>
                <p>Jing Guo, Hao Chen, Peng Yang, Yew  Ti Lee, Min Wu, Teresa M. Przytycka, Chee Keong Kwoh, <strong>Jie Zheng*</strong>. LDSplitDB: A database for studies of meiotic  recombination hotspots in MHC using human genomic data. <em>BMC Medical Genomics </em>11(Suppl 2): 27, special issue of the 28th  International Conference on Genome Informatics (GIW/BIOINFO 2017), 2018 (<strong>IF</strong> = 3.317).<br>
                  Website of LDSplitDB: <a href="http://histone.scse.ntu.edu.sg/LDSplitDB/">http://histone.scse.ntu.edu.sg/LDSplitDB/</a></p>
              </li>
              <li>
                <p>Huey Eng Chua, Sourav S. Bhowmick*, <strong>Jie Zheng</strong>. <a href="https://academic.oup.com/bioinformatics/article-abstract/doi/10.1093/bioinformatics/btx600/4209992?redirectedFrom=fulltext">TROVE: A User-friendly Tool  for Visualizing and Analyzing Cancer Hallmarks in Signaling Networks</a>. <em>Bioinformatics</em>, 34(2):314-316, 2018 (<strong>IF</strong> = 5.481).</p>
              </li>
              <li>
                <p>Hui Liu, Libo Luo, Zhanzhan Cheng,  Jianjiang Sun, Jihong Guan, <strong>Jie Zheng*</strong>,  Shuigeng Zhou* (co-corresponding authors). Group-sparse modeling drug-kinase  networks for predicting combinatorial drug sensitivity in cancer cells. <em>Current Bioinformatics</em>, Vol. 13 (E-pub  ahead of print), 2018 (<strong>IF</strong> = 0.540, <strong>5-year IF</strong> = 0.777).</p>
              </li>
              <li>
                <p>Jing Guo, Hao Chen, Peng Yang, Yew  Ti Lee, Min Wu, Teresa M. Przytycka, Chee Keong Kwoh, <strong>Jie Zheng</strong>. LDSplitDB: A database for studies of meiotic  recombination hotspots in MHC using human genomic data. <em>BMC Medical Genomics </em>(special issue of the 28th  International Conference on Genome Informatics (GIW/BIOINFO 2017)), 2017, in  press (<strong>IF</strong> = 2.448).</p>
              </li>
            </ul>
            <p>&nbsp;</p>
            <p><strong>2017</strong></p>
            <ul>
              <li>
                <p>Lichun Ma, <strong>Jie Zheng</strong>. A polynomial based model for cell fate prediction in  human diseases. <em>BMC Systems Biology</em> (special issue of the 16th International Conference on  Bioinformatics (InCoB 2017)), 2017, in press (<strong>IF</strong> = 2.303). (<strong>Best Paper Award,  Gold Medal</strong>). <strong>&nbsp;</strong></p>
              </li>
              <li>
                <p>Jing Guo, <strong>Jie Zheng</strong>. <a href="https://academic.oup.com/bioinformatics/article/33/14/i102/3953947">HopLand: Single-cell pseudotime  recovery using continuous Hopfield network based modeling of Waddington&rsquo;s  epigenetic landscape</a>, <em>Bioinformatics</em>,  33(14): i102 &ndash; i109 (special issue of conference ISMB/ECCB 2017, acceptance  rate 16.5%), 2017 (<strong>IF</strong> = 7.307).<strong> </strong><br>
                Source code of HopLand: <a href="https://github.com/NetLand-NTU/HopLand/">https://github.com/NetLand-NTU/HopLand/</a> </p>
              </li>
              <li>
                <p>M  Pek, SMJM Yatim, Y Chen, J Li, M Gong, X Jiang, F Zhang*, <strong>J Zheng</strong>, X Wu, Q Yu, <a href="https://www.nature.com/articles/onc2017120">Oncogenic  KRAS-associate gene signature defines co-targeting of CDK4/6 and MEK as a  viable therapeutic strategy in colorectal cancer</a>. <em>Nature </em><em>Oncogene</em>, 2017, in press (<strong>IF</strong> = 7.519).</p>
              </li>
              <li>                Jing  Guo, Feng Lin, Xiaomeng Zhang, Vivek Tanavde, <strong>Jie Zheng</strong>. <a href="https://academic.oup.com/bioinformatics/article-abstract/doi/10.1093/bioinformatics/btx022/2929342/NetLand-quantitative-modeling-and-visualization-of?redirectedFrom=fulltext">NetLand: quantitative modeling and visualization of  Waddington&rsquo;s epigenetic landscape using probabilistic potential</a>. <em>Bioinformatics</em>, <strong>33</strong>(10), pp. 1583 - 1585, 2017  (<strong>IF</strong> = 7.307).<br>
              Software source code and user manual: <a href="http://netland-ntu.github.io/NetLand/">https://github.com/NetLand-NTU/NetLand/</a></li>
            </ul>
            <p>&nbsp;</p>
            <p><strong>2016</strong></p>
            <ul>
              <li>
                <p>Jing Guo, Hui Liu, <strong>Jie Zheng</strong>. <a href="https://academic.oup.com/nar/article/44/D1/D1011/2502617/SynLethDB-synthetic-lethality-database-toward">SynLethDB:  synthetic lethality database toward discovery of selective and sensitive  anticancer drug targets</a>. <em>Nucleic Acids Research</em>, 44 (D1), D1011 - 1017 (Database issue, open access on-line), 2016 (<strong>IF</strong> = 10.162).<br>
                  SynLethDB website: <a href="http://histone.sce.ntu.edu.sg/SynLethDB/">http://histone.scse.ntu.edu.sg/SynLethDB/</a> </p>
              </li>
              <li>
                <p>Hui Liu, Fan Zhang, Shital Kumar Mishra, Shuigeng Zhou, <strong>Jie Zheng</strong>. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5075921/">Knowledge-guided fuzzy logic modeling to infer cellular signaling networks from proteomic data</a>. <em>Scientific Reports</em>, 6:35652, 2016 (<strong>IF</strong> = 5.228).<br>
                  Source code and dataset: <a href="http://www.ntu.edu.sg/home/zhengjie/software/fuzzySigNet/">http://www.ntu.edu.sg/home/zhengjie/software/fuzzySigNet/</a></p>
              </li>
              <li>
                <p>Fan Zhang, Min Wu, Chee Keong Kwoh, <strong>Jie Zheng</strong>. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0165049">Power-law modeling of cancer cell fates driven by signaling data to reveal drug effects</a>. <em>PLOS ONE</em>, 11(10):e0165049, 2016 (<strong>IF</strong> = 3.54). </p>
              </li>
              <li>
                <p>Haifen Chen, Xinrui Zhou, <strong>Jie Zheng</strong>, Chee Keong Kwoh. <a href="https://bmcmedgenomics.biomedcentral.com/articles/10.1186/s12920-016-0230-5">Rules of co-occurring mutations characterize the antigenic evolution of human influenza A/H3N2, A/H1N1 and B viruses</a>. <em>BMC Medical Genomics</em>, 9:230, 2016 (special issue of the 15th International Conference on Bioinformatics, INCOB 2016, <strong>IF</strong> = 2.726).</p>
              </li>
              <li>
                <p>Fan Zhang, Runsheng Liu, <strong>Jie Zheng</strong>. <a href="https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-016-0365-1">Sig2GRN: a software tool linking signaling pathway with gene regulatory network for dynamic simulation</a>. <em>BMC Systems Biology</em>, 10(4), 541, 2016 (special issue of the 27th International Conference on Genome Informatics, GIW 2016, <strong>IF</strong> = 2.65).<br>
                  Source  code: <a href="http://histone.scse.ntu.edu.sg/Sig2GRN/">http://histone.scse.ntu.edu.sg/Sig2GRN/</a></p>
              </li>
              <li>
                <p>Fan Zhang, Haoting Chen, Li Na  Zhao, Hui Liu, Teresa M. Przytycka and <strong>Jie  Zheng</strong>. <a href="https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-015-0249-9">Generalized logical model based on network topology to capture the  dynamical trends of cellular signaling pathways</a>. <em>BMC Systems Biology</em>, 10 Suppl 1:7, 2016 (special issue of APBC 2016, <strong>IF</strong> = 2.65).</p>
              </li>
              <li>                Yifeng Li*, Haifen Chen* (*joint first authors), <strong>Jie  Zheng</strong>, Alioune Ngom. <a href="http://ieeexplore.ieee.org/abstract/document/7229306/?reload=true">The  Max-Min High-Order Dynamic Bayesian Network for Learning Gene Regulatory  Networks with Time-Delayed Regulations</a>. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em> (IEEE/ACM  TCBB), Vol. 13, Issue 4, pp. 792 - 803, 2016 (<strong>IF</strong> =  1.438).</li>
            </ul>
            <p>&nbsp;</p>
            <p><strong>2015 and before </strong></p>
            <ul>
              <li>
                <p>Hao Chen, Peng Yang, Jing Guo, Chee Keong Kwoh, Teresa M. Przytycka, <strong>Jie Zheng</strong>. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-16-S12-S1">ARG-walker: Inference of Individual Specific Strengths of Meiotic  Recombination Hotspots by Population Genomics Analysis</a>. <em>BMC Genomics</em>, 16(Suppl 12):S1, 2015 (<strong>IF</strong> = 3.99).</p>
              </li>
              <li>
                <p>Haifen  Chen, Jing Guo, Shital K Mishra, Paul Robson, Mahesan Niranjan, <strong>Jie Zheng</strong>. <a href="http://bioinformatics.oxfordjournals.org/content/31/7/1060">Single-cell transcriptional analysis to uncover regulatory  circuits driving cell fate decisions in early mouse development</a>. <em>Bioinformatics</em>, 31(7), pp. 1060 &ndash;  1066, 2015 (<strong>IF</strong> = 5.766). </p>
              </li>
              <li>
                <p>Hui Liu, Jianjiang Sun, Jihong Guan, <strong>Jie Zheng</strong> and Shuigeng Zhou. <a href="https://bioinformatics.oxfordjournals.org/content/31/12/i221.full">Improving  compound&ndash;protein interaction prediction by building up highly credible negative  samples</a>. <em>Bioinformatics</em> 31  (12), i221 &ndash; i229, (special issue of ISMB/ECCB 2015), 2015 (<strong>IF</strong> = 5.766, <strong>5-year IF</strong> =  6.968). </p>
              </li>
              <li>
                <p>Shital K  Mishra, Sourav S Bhowmick, Huey Eng Chua, Fan Zhang, <strong>Jie Zheng</strong>. <a href="http://www.biomedcentral.com/content/pdf/1752-0509-9-S1-S4.pdf">Computational cell fate  modelling for discovery of rewiring in apoptotic network for enhanced cancer  drug sensitivity</a>. <em>BMC  Systems Biology</em>, 9 (Suppl. 1):S4, 2015 (presented at APBC 2015,<strong> IF</strong> = 2.853). </p>
              </li>
              <li>
                <p>Mar&iacute;a Mu&ntilde;oz&#8208;Amatria&iacute;n, Stefano Lonardi,  &hellip;, <strong>Jie Zheng</strong>, &hellip;, Timothy J Close. <a href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.12959/full">Sequencing  of 15 622 gene&#8208;bearing  BACs clarifies the gene&#8208;dense  regions of the barley genome</a>. <em>The Plant Journal</em>, Volume 84, Issue 1, pages 216 &ndash; 227, 2015 (open  access, <strong>IF</strong> =&nbsp; 5.972).</p>
              </li>
              <li>
                <p>Fan Zhang,  Min Wu, Xuejuan Li, Xiaoli Li, Chee Keong Kwoh, <strong>Jie Zheng</strong>. <a href="http://dx.doi.org/10.1142/S0219720015410024">Predicting essential  genes and synthetic lethality via influence propagation in signaling pathways  of cancer cell fates</a>. <em>Journal of Bioinformatics and  Computational Biology</em>,  Vol. 13, No. 03, 1541002 (14 pages), 2015. </p>
              </li>
              <li>
                <p>Li Na Zhao, <strong>Jie Zheng</strong>, Lock  Yue Chew, Yuguang Mu. <a href="http://pubs.acs.org/doi/abs/10.1021/acs.jpcb.5b00957">An  Investigation on the Fundamental Interaction between Abeta Peptides and the  AT-Rich DNA</a>. <em>The Journal of  Physical Chemistry B</em>, Volume 119, Issue 26, Pages 8247 &ndash; 8259, 2015 (IF =  3.302). </p>
              </li>
              <li>
                <p>Hongliang Hu, Qinghua Zhang, Shen Li, Xiaona Zhang, Junsong Han,  Huasheng Xiao, David Yan, <strong>Jie Zheng</strong>,  Biaoru Li. <a href="http://www.researchgate.net/profile/Biaoru_Li/publication/273772027_A_Therapeutic_Targeting_Identification_from_Microarray_Data_and_Quantitative_Network_Analysis-A_Case_report_of_Advanced_Breast_Cancer/links/550c491e0cf275261095bfcb.pdf">A  Therapeutic Targeting Identification from Microarray Data and Quantitative  Network Analysis</a>. <em>The Open Access Journal of Science &amp; Technology</em>, Volume 3,  Article ID 101114, Pages 10, 2015.</p>
              </li>
              <li>
                <p>Jing  Guo, Ritika Jain, Peng Yang, Rui Fan, Chee-Keong Kwoh, <strong>Jie Zheng</strong>. <a href="https://www.computer.org/csdl/trans/tb/preprint/06646172.pdf">Reliable and Fast Estimation of Recombination Rates by  Convergence Diagnosis and Parallel Markov Chain Monte Carlo</a>. <em>ACM/IEEE Transactions on Computational  Biology and Bioinformatics</em> (TCBB), 11(1), pp. 1-10, 2014 (presented at GIW 2013, <strong>IF = </strong>1.62).</p>
              </li>
              <li>
                <p>Min Wu,  Chee-Keong Kwoh, Xiaoli Li, <strong>Jie Zheng</strong>. <a href="http://www.biomedcentral.com/1752-0509/8/107">Finding trans-regulatory genes and protein complexes  modulating meiotic recombination hotspots of human, mouse and yeast</a>. <em>BMC Systems Biology</em>, 8:107, 2014 (<strong>IF</strong> = 2.853)<strong>.</strong></p>
              </li>
              <li>
                <p>Haifen  Chen, Piyushkumar A Mundra, Li Na Zhao, Feng Lin, <strong>Jie Zheng</strong>. <a href="http://www.biomedcentral.com/1752-0509/8/S4/S6">Highly sensitive  inference of time-delayed gene regulation by network deconvolution</a>. <em>BMC Systems Biology</em>, 8 (Suppl. 4),  S6, 2014 (special issue of InCoB 2014, <strong>IF</strong> = 2.853)<strong>.</strong></p>
              </li>
              <li>
                <p>Peng Yang, Min Wu, Jing Guo, Chee  Keong Kwoh, Teresa M Przytycka, <strong>Jie  Zheng</strong>. <a href="http://www.biomedcentral.com/1471-2105/15/48">LDsplit:  screening for cis-regulatory motifs stimulating meiotic recombination hotspots  by analysis of DNA sequence polymorphisms</a>. <em>BMC  Bioinformatics</em>, <strong>15</strong>:48, 2014 (<strong>IF</strong> = 2.67)<strong>.</strong></p>
              </li>
              <li>
                <p>Wenting Liu, Kui Miao, Guangxia  Li, Kuiyu Chang, Jie Zheng, Jagath C Rajapakse. <a href="http://www.biomedcentral.com/1471-2105/15/S2/S4">Extracting rate  changes in transcriptional regulation from MEDLINE abstracts</a>. <em>BMC Bioinformatics</em>, <strong>15</strong>(Suppl 2):S4, 2014 (presented at APBC 2014, <strong>IF</strong> = 2.67).</p>
              </li>
              <li>
                <p>Chinh Tran-To Su, Thuy-Diem Nguyen, Jie Zheng,  Chee Keong Kwoh. <a href="http://www.biomedcentral.com/1471-2105/15/S16/S9">IFACEwat: the  interfacial water-implemented re-ranking algorithm to improve the  discrimination of near native structures for protein rigid docking</a>. <em>BMC Bioinformatics</em>, 15 (Suppl. 16),  S9, 2014. (presented at InCoB 2014, <strong></strong><strong>IF</strong> = 2.67).</p>
              </li>
              <li>
                <p>Ragini Rai, Lei Zhu, Haifen Chen*, Archana  Patkar Gupta, Siu Kwan Sze, Jie Zheng, Christiane Ruedl, Zbynek Bozdech, Mark  Featherstone. <a href="http://www.biomedcentral.com/1471-2164/15/959">Genome-wide analysis in Plasmodium  falciparum reveals early and late phases of RNA polymerase II occupancy during  the infectious cycle</a>. <em>BMC  Genomics</em>, 15:959, 2014 (<strong>IF</strong> =  4.04).</p>
              </li>
              <li>
                <p>Xue-juan Li, Shital K  Mishra, Min Wu, Fan Zhang, <strong>Jie Zheng</strong>. <a href="http://www.hindawi.com/journals/bmri/2014/196034/">Syn-Lethality: An Integrative Knowledge Base of Synthetic Lethality towards  Discovery of Selective Anticancer Therapies</a>. <em>BioMed Research International</em>, Vol. 2014, Article ID 196034, 7  pages, 2014 (<strong>IF</strong> = 2.706).</p>
              </li>
              <li>
                <p>Haifen  Chen, Stefano Lonardi, <strong>Jie Zheng</strong>. <a href="http://www.sciencedirect.com/science/article/pii/S1476927114000048">Deciphering  Histone Code of Transcriptional Regulation in Malaria Parasites by Large-scale  Data Mining</a>. <em>Computational Biology and Chemistry</em>, Vol. 50, pp. 3-10, 2014 (presented at APBC 2014, <strong>IF</strong> = 1.595).</p>
              </li>
              <li>
                <p>Min Wu, Xuejuan Li, Fan Zhang, Xiaoli Li,  Chee-Keong Kwoh, <strong>Jie Zheng</strong>. <a href="http://www.la-press.com/in-silico-prediction-of-synthetic-lethality-by-meta-analysis-of-geneti-article-a4492-abstract?article_id=4492">In Silico Prediction of Synthetic Lethality by Meta-Analysis  of Genetic Interactions, Functions, and Pathways in Yeast and Human Cancer</a>. <em>Cancer Informatics</em>, 13(Suppl. 3), pp.  71-80, 2014 (<strong>IF</strong> = 1.36). </p>
              </li>
              <li>
                <p>Pamela Thompson, Kevin Urayama, Jie Zheng, Peng  Yang, Matt Ford, Patricia Buffler, Anand Chokkalingam, Tracy Lightfoot,  Malcolm Taylor. <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0100480">Differences in meiotic  recombination rates in childhood acute lymphoblastic leukemia at an MHC class  II hotspot close to disease associated haplotypes</a>. <em>PLOS ONE</em>, 9(6):e100480, 2014 (<strong>IF</strong> = 3.534). </p>
              </li>
              <li>
                <p>Yunbo Xu, Hongliang  Hu, <strong>Jie Zheng</strong>#, Biaoru Li# (co-corresponding author). <a href="http://www.hindawi.com/journals/gri/2013/724124/abs/">Feasibility  of Whole RNA Sequencing from Single-Cell mRNA Amplification</a>, <em>Genetics  Research International</em>,  Vol. 2013, No. 724124, 2013. </p>
              </li>
              <li>
                <p>M. Wu, Q. Yu, X-L, Li, <strong>J. Zheng</strong>, J-F Huang, C-K Kwoh. <a href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0053197">Benchmarking Human Protein Complexes to Investigate Drug-Related Systems and Evaluate Predicted Protein Complexes</a>. <em>PLOS ONE</em> 8(2): e53197, 2013 (<strong>IF</strong> = 4.092).</p>
              </li>
              <li>
                <p>J. Mei, C-K Kwoh, P. Yang, X-L Li, <strong>J.  Zheng</strong>. <a href="http://bioinformatics.oxfordjournals.org/content/29/2/238.short">Drug-Target Interaction Prediction by Learning from Local Information and Neighbors</a>. <em>Bioinformatics</em>,  29(2): 238-245, 2013 (<strong>IF</strong> = 5.766) (<a href="publications/Mei_Bioinformatics_2013.pdf">PDF</a>).</p>
              </li>
              <li>
                <p>Daniel Lin-Kit Wong, Xiao-Li Li, Min Wu, <strong>Jie Zheng</strong>, See-Kiong Ng. <a href="http://link.springer.com/article/10.1186/1471-2164-14-S5-S15">PLW: Probabilistic  Local Walks for detecting protein complexes from protein interaction networks</a>. <em>BMC Genomics</em>, <strong>14</strong>(Suppl 5): S15, 2013 (<strong>IF</strong> = 4.40, conference version presented at Twelfth International Conference on  Bioinformatics (InCoB2013), Taicang, China, Sept. 20-22, 2013).</p>
              </li>
              <li>
                <p>Chinh Tran-To Su, Xuchang Ouyang, <strong>Jie Zheng</strong>, Chee-Keong Kwoh. <a href="http://www.biomedcentral.com/1471-2105/14/S16/S7">Structural analysis of the  novel influenza A (H7N9) viral Neuraminidase interactions with current approved  neuraminidase inhibitors Oseltamivir, Zanamivir, and Peramivir in the presence  of mutation R289K</a>. <em>BMC  Bioinformatics</em>, <strong>14</strong> (Suppl  16), S7, 2013 (<strong>IF</strong> = 3.02,  conference version presented at Twelfth International Conference on Bioinformatics (InCoB2013), Taicang,  China, Sept. 20-22, 2013).</p>
              </li>
              <li>
                <p>Min Wu, Zhipeng Xie, Xiaoli Li, Chee&#8208;Keong  Kwoh, <strong>Jie Zheng</strong>. <a href="http://onlinelibrary.wiley.com/doi/10.1002/prot.24365/full">Identifying  protein complexes from heterogeneous biological data</a>. <em>Proteins: Structure, Function, and Bioinformatics,</em> 81(11),  2023-2033, 2013 (<strong>IF</strong> = 3.337).</p>
              </li>
              <li>
                <p>R. Salari#, D. Wojtowicz#, <strong>J. Zheng</strong># (joint first authors in alphabetical order), D. Levens. Y. Pilpel, TM Przytycka. <a href="http://www.ploscompbiol.org/article/info%3Adoi%2F10.1371%2Fjournal.pcbi.1002644">Teasing Apart Translational and Transcriptional Components of Stochastic Variations in Eukaryotic Gene Expression</a>. <em>PLoS Computational Biology</em>, 8(8):e1002644, 2012 (<strong>IF</strong> = 5.515).</p>
              </li>
              <li>
                <p>Min Wu, Chee-Keong Kwoh,  Teresa Przytycka, Jing Li, and <strong>Jie  Zheng</strong>. <a href="http://www.proteomesci.com/content/10/S1/S11">Epigenetic Functions Enriched in Transcription Factors Binding to  Mouse Recombination Hotspots</a>. <em>Proteome  Science</em>, 10(Suppl 1):S11, 2012 (<strong>IF</strong> = 2.42).</p>
              </li>
              <li>
                <p><strong class="STYLE7">J. Zheng</strong>, T. Jiang. <a href="http://www.ccf.org.cn/sites/ccf/tybgcont.jsp?contentId=2590507188096">International Conference on Research in Computational Molecular Biology</a>. <em>Communications of China Computer Federation</em> (CCCF), vol. 59, 2011 (<a href="publications/Zheng_cccf_recomb_2011.pdf">PDF</a>). </p>
              </li>
              <li>
                <p><strong>J.  Zheng</strong>, PP Khil, RD  Camerini-Otero, TM Przytycka. <a href="http://genomebiology.com/2010/11/10/R103"></a><a href="http://genomebiology.com/2010/11/10/R103">Detecting  sequence polymorphisms associated with meiotic recombination hotspots in the human  genome</a>. <em>Genome Biology</em>, 11:R103. 2010<span class="STYLE6"><span class="STYLE7"> <img src="images/bmc_highlyaccessed.png" alt="bmc highly access" width="130" height="24" align="middle"> </span></span>(Highlighted on <a href="http://www.genomeweb.com//node/955844?hq_e=el&hq_m=873294&hq_l=5&hq_v=4f1fc1f8ae">daily news of GenomeWeb</a>) (<strong>IF</strong> = 11.313).</p>
              </li>
              <li>
                <p><strong>J.  Zheng*</strong>, D. Zhang, PF  Przytycki, R. Zielinski, J. Capala, TM Przytycka* (co-corresponding author). <a href="http://bioinformatics.oxfordjournals.org/content/26/1/141.abstract">SimBoolNet - A Cytoscape  plugin for dynamic simulation of signaling networks</a>. <em>Bioinformatics</em>,  26(1): 141-142, 2010 (<strong>IF</strong> = 5.766).</p>
              </li>
              <li>
                <p>R. Zielinski, PF Przytycki, <strong>J. Zheng</strong>,  D. Zhang, TM Przytycka, J. Capala. <a href="http://www.biomedcentral.com/1752-0509/3/88/">The crosstalk between EGF, IGF, and Insulin cell signaling  pathways &ndash; computational and experimental analysis</a>. <em>BMC Systems Biology</em>, 3(88), 2009 <img src="images/bmc_highlyaccessed.png" alt="bmc highly access" width="130" height="24" align="middle"> (<strong>IF</strong> = 3.148).</p>
              </li>
              <li>
                <p><strong>J. Zheng</strong>, IB Rogozin, EV Koonin, TM Przytycka. <a href="http://mbe.oxfordjournals.org/content/24/11/2583.long">Support for the Coelomata Clade of animals from a  rigorous analysis of the pattern of intron conservation</a>. <em>Molecular  Biology and Evolution</em>,  24(11):2583-92, 2007 (<strong>IF</strong> = 13.649).</p>
              </li>
              <li>
                <p><strong>J. Zheng</strong>, J. Svensson, K.  Madishetty, T. Close, T. Jiang, S. Lonardi. <a href="http://www.biomedcentral.com/1471-2105/7/7/">OligoSpawn: a software tool  for the design of overgo probes from large unigene databases</a>. <em>BMC  Bioinformatics</em>,  7:7, 2006 (<strong>IF</strong> = 2.751).</p>
              </li>
              <li>
                <p>X. Chen, <strong>J. Zheng</strong>, Z. Fu, P. Nan, Y. Zhong, S.  Lonardi and T. Jiang. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17044168">Assignment  of orthologous genes via genome rearrangement</a>. <em>IEEE/ACM Transactions  on Computational Biology and Bioinformatics</em>, 2(4), 302-315, 2005 (<strong>IF</strong> = 1.543).</p>
              </li>
              <li>
                <p>A. Goldstein, P. Kolman, <strong>J. Zheng</strong> (authors in alphabetical order). <a href="http://www.combinatorics.org/Volume_12/Abstracts/v12i1r50.html">Minimum  common string partition problem: hardness and approximation</a>. <em>Electronic Journal of  Combinatorics</em>, 12(1), 2005.</p>
              </li>
              <li>
                <p><strong>J. Zheng</strong>, T. Close, T. Jiang,  S. Lonardi. <a href="http://bioinformatics.oxfordjournals.org/content/20/13/2101.long">Efficient  Selection of Unique and Popular Oligos for Large EST Databases</a>. <em>Bioinformatics</em>,  20(13), 2101-2112, 2004 (<strong>IF</strong> = 5.766). </p>
              </li>
            </ul>
            <p class="STYLE12 STYLE7 STYLE11 STYLE11 STYLE11 STYLE11 STYLE11">&nbsp;</p>
            <p class="STYLE12 STYLE7 STYLE11 STYLE11 STYLE11 STYLE11 STYLE11"><span class="STYLE6 STYLE11"><strong>Conferences </strong></span></p>
            <ol>
              <li>
                <p>Jing Guo, Hao Chen, Peng Yang, Yew  Ti Lee, Min Wu, Teresa M. Przytycka, Chee Keong Kwoh, <strong>Jie Zheng</strong>. LDSplitDB: A database for studies of meiotic  recombination hotspots in MHC using human genomic data. <em>The 28th International Conference on Genome Informatics</em> (GIW/BIOINFO 2017)), Seoul, Korea, 31 Oct. &ndash; 3 Nov. 2017. </p>
              </li>
              <li>
                <p>Lichun Ma, <strong>Jie Zheng</strong>. A polynomial based model for cell fate prediction in  human diseases. <em>The 16th  International Conference on Bioinformatics</em> (InCoB 2017), Shenzhen, China,  20 &ndash; 22 Sept., 2017.</p>
              </li>
              <li>
                <p>Fransiskus Xavier Ivan, Xinrui Zhou,  Akhila Deshpande, Rui Ying, <strong>Jie Zheng</strong>, Chee Keong Kwoh. Phylogenetic tree based  method for uncovering co-mutational site-pairs in influenza viruses. <em>Proc. of 8th ACM Conference on  Bioinformatics, Computational Biology, and Health Informatics</em> (ACM-BCB),  pages 21 &ndash; 26, Boston, MA, USA, 20 &ndash; 23 August, 2017.</p>
              </li>
              <li>
                <p>Jing Guo, <strong>Jie Zheng</strong>.  HopLand: Single-cell pseudotime recovery using continuous Hopfield network  based modeling of Waddington&rsquo;s epigenetic landscape. <em>Proc. Intelligent Systems for Molecular Biology and European Conference  on Computational Biology (ISMB/ECCB)</em> 2017, to appear, Prague, Czech  Republic, 21 &ndash; 25 July, 2017. [<strong>acceptance  rate 16.5%</strong>].</p>
              </li>
              <li>
                <p>Rui Yin, Xinrui Zhou, Fransiskus  Xavier Ivan, <strong>Jie Zheng</strong>, Vincent T. K. Chow, Chee Keong Kwoh. Identification of  potential critical virulent sites based on hemagglutinin of influenza A virus  in past pandemic strains. <em>Proc. 6th  International Conference on Bioinformatics and Biomedical Science</em> (ICBBS),  Singapore, June 22 &ndash; 24, 2017.</p>
              </li>
              <li>
                <p>Shital Kumar Mishra, Sourav S Bhowmick, Huey Eng  Chua and <strong>Jie Zheng</strong>. <a href="http://dl.acm.org/citation.cfm?id=2975188">Predictive  Modeling of Drug Effects on Signaling Pathways in Diverse Cancer Cell Lines</a>. <em>The 7th ACM Conference on  Bioinformatics, Computational Biology, and Health Informatics</em> (ACM-BCB  2016), pp. 200 &ndash; 209, Seattle, WA, USA, 2 &ndash; 5 Oct. 2016.</p>
              </li>
              <li>
                <p>Huey Eng Chua, Sourav S Bhowmick, Jie Zheng and Lisa Tucker-Kellogg.  <a href="http://dl.acm.org/citation.cfm?id=2975178">TAPESTRY: Network-centric Target Prioritization in Disease-related Signaling  Networks</a>. <em>The 7th ACM  Conference on Bioinformatics, Computational Biology, and Health Informatics</em> (ACM-BCB 2016), pp. 108 &ndash; 117, Seattle, WA, USA, 2 &ndash; 5 Oct. 2016.</p>
              </li>
              <li>
                <p>Hui Liu, Zhanzhan Cheng, Jiangjiang Sun, Jihong Guan, Jie Zheng,  Shuigeng Zhou. Sparse linear modeling kinase inhibition network for predicting  combinatorial drug sensitivity in cancer cells. <em><a href="http://admis.fudan.edu.cn/cbc2016/">The first CCF Bioinformatics Conference</a></em> (CBC 2016), Chongqing,  China, 10 &ndash; 13 Nov. 2016.</p>
              </li>
              <li>
                <p>Haifen Chen, Xinrui Zhou, Jie  Zheng, Chee Keong Kwoh. Rules of co-occurring mutations characterize the  antigenic evolution of human influenza A/H3N2, A/H1N1 and B viruses. <em>Proc. <a href="https://incob.apbionet.org/incob16/#">15th International  Conference on Bioinformatics</a></em> (INCOB 2016), Biopolis, Singapore, 21st  &ndash; 23rd Sept. 2016.</p>
              </li>
              <li>
                <p>Fan Zhang*, Runsheng Liu*, <strong>Jie  Zheng</strong>. Sig2GRN: a software tool linking signaling pathway with gene  regulatory network for dynamic simulation. <a href="http://admis.fudan.edu.cn/giw2016/"><em>The  27th International Conference on Genome Informatics</em></a> (GIW2016), Shanghai, China, 3 &ndash; 5 Oct.  2016. </p>
              </li>
              <li>
                <p>Fan Zhang, Haoting Chen, Li Na Zhao, Hui Liu,  Teresa M. Przytycka and <strong>Jie Zheng</strong>.  Generalized logical model based on network topology to capture the dynamical  trends of cellular signaling pathways. <em>Proc.</em> <a href="http://www.sfasa.org/apbc2016/apbc2016.html">14th <em>Asia Pacific Bioinformatics  Conference</em></a> (APBC  2016), San Francisco, California, USA, 11 &ndash; 13 January, 2016.</p>
              </li>
              <li>
                <p>Hao  Chen, Peng Yang, Jing Guo, Chee Keong Kwoh, Teresa M. Przytycka, <strong>Jie Zheng</strong>. ARG-walker: Inference of Individual Specific Strengths of Meiotic  Recombination Hotspots by Population Genomics Analysis. <a href="http://www.jsbi.org/giw-incob2015/"><em>The 26th Genome Informatics Workshop and 14th  International Conference on Bioinformatics (GIW/InCoB 2015)</em></a>, Odaiba, Tokyo, Japan, 9 &ndash; 11 Sept. 2015.</p>
              </li>
              <li>
                <p>Hui Liu, Jianjiang Sun, Jihong Guan, <strong>Jie Zheng</strong> and Shuigeng Zhou. <a href="https://bioinformatics.oxfordjournals.org/content/31/12/i221.full">Improving  compound&ndash;protein interaction prediction by building up highly credible negative  samples</a>. <a href="https://www.iscb.org/ismbeccb2015"><em>The 23rd  Annual International Conference on Intelligent Systems for Molecular Biology  &amp; 14th European Conference on Computational Biology (ISMB/ECCB  2015)</em></a>, Dublin, Ireland, 10 &ndash; 14, July, 2015.</p>
              </li>
              <li>
                <p>Haifen Chen, Jing  Guo, Shital K Mishra, Paul Robson, Mahesan Niranjan, <strong>Jie Zheng</strong>. Single-cell  transcriptional analysis to uncover regulatory circuits driving cell fate  decisions in early mouse development. <a href="http://www.c-bio.org/2015/"><em>The 2nd Symposium on  Complex Biodynamics &amp; Networks (cBio 2015)</em></a>,  Tsuruoka, Japan, 11 &ndash; 13 May 2015.</p>
              </li>
              <li>
                <p>Shital K Mishra, Sourav S Bhowmick, Huey Eng Chua, Fan Zhang, <strong>Jie Zheng</strong>. Computational cell fate modelling for discovery of rewiring in apoptotic  network for enhanced cancer drug sensitivity. <a href="http://apbc2015.mbc.nctu.edu.tw/"><em>The 13th  Asia Pacific Bioinformatics Conference (APBC 2015)</em></a>, HsinChu, Taiwan, China,  21 &ndash; 23 January 2015.</p>
              </li>
              <li>
                <p>Fan Zhang, Min Wu,  Xuejuan Li, Xiaoli Li, Chee Keong Kwoh, <strong>Jie  Zheng</strong>. Predicting essential genes and  synthetic lethality via influence propagation in signaling pathways of cancer  cell fates. <em><a href="http://www.csbio.org/2014/">The 5th International Conference on Computational Systems Biology and Bioinformatics (CSBio 2014)</a>, </em>Singapore, 10 - 12 Nov. 2014.</p>
              </li>
              <li>
                <p>Fan Zhang, Chee-Keong Kwoh, Min Wu, <strong>Jie Zheng</strong>. <a href="http://dl.acm.org/citation.cfm?id=2649425">Data-driven prediction of cancer cell fates with a  nonlinear model of signaling pathways.</a> <em>Proc.  5th ACM Conference on Bioinformatics, Computational Biology, and  Health Informatics</em> (ACM-BCB2014), pp. 436 &ndash; 444, Newport Beach, California,  USA, 20 &ndash; 23 Sept. 2014.</p>
              </li>
              <li>
                <p>Seyed Ziaeddin Alborzi, DAK Maduranga, Rui Fan, Jagath C Rajapakse, <strong>Jie Zheng</strong>. <a href="http://link.springer.com/chapter/10.1007/978-3-319-09192-1_8">CUDAGRN: Parallel Speedup of  Inferring Large Gene Regulatory Networks from Expression Data Using Random  Forest</a>. <em>Proc. 9th IAPR  International Conference of Pattern Recognition in Bioinformatics</em> (PRIB2014),  LNCS 8626, pp. 85 &ndash; 97, Stockholm, Sweden, 21 &ndash; 23 Aug. 2014.</p>
              </li>
              <li>
                <p>Sourav  S Bhowmick, Huey-Eng Chua, <strong>Jie Zheng</strong>. <a href="http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=6741401&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D6741401">INGOT: Towards Network-driven In Silico  Combination Therapy</a>. <em>Proc. of 1st  International Conference on Big Data and Smart Computing</em> (BigComp 2014), Bangkok, Thailand, 15 &ndash; 17 Jan. 2014.</p>
              </li>
              <li>
                <p>Xue-juan Li, Shital Kumar Mishra, Min Wu, Fan Zhang, and <strong>Jie Zheng</strong>. Syn-Lethality: An  Integrative Knowledge Base of Synthetic Lethality towards Discovery of  Selective Anticancer Therapies.<em> IEEE International Conference on Bioinformatics  and Biomedicine</em> (BIBM), Workshop &quot;<em>Biomolecular Networks and Human Diseases</em>&quot;, Shanghai, China, 2013.  (<a href="publications/Li_BIBM_2013_SLDB.pdf">PDF</a>)</p>
              </li>
              <li>
                <p>Min Wu, Xuejuan Li, Fan Zhang,  Xiaoli Li, Chee-Keong Kwoh, <strong>Jie Zheng</strong>. <a href="http://dl.acm.org/citation.cfm?id=2506653">Meta-analysis of Genomic and Proteomic Features to  Predict Synthetic Lethality of Yeast and Human Cancer</a>. Proc. the  International Conference on Bioinformatics, Computational Biology and  Biomedical Informatics (ACM-BCB), pp. 384-  391, 2013. (<a href="publications/Wu_ACM-BCB_2013_MetaSL.pdf">PDF</a>) </p>
              </li>
              <li>
                <p>D.A.K. Maduranga, <strong>Jie Zheng</strong>, Piyushkumar A. Mundra, Jagath C. Rajapakse. <a href="http://link.springer.com/chapter/10.1007/978-3-642-39159-0_2">Inferring gene regulatory networks from time-series  expressions using random forests ensembl</a>. <em>Proc. Pattern Recognition in  Bioinformatics</em> (PRIB), LNBI 7986, pp. 13-22, 2013. (<a href="publications/Kasun_PRIB_2013_RanfomForests.pdf">PDF</a>)</p>
              </li>
              <li>
                <p>Piyushkumar A. Mundra, <strong>Jie Zheng</strong>, Mahesan Niranjan, Roy E.  Welsch, Jagath C. Rajapakse. <a href="http://scholar.google.com/citations?view_op=view_citation&hl=en&user=evC66XkAAAAJ&sortby=pubdate&citation_for_view=evC66XkAAAAJ:_Qo2XoVZTnwC">Inferring Time-Delayed Gene Regulatory Networks Using  Cross-Correlation and Sparse Regression</a>. <em>Proc. Bioinformatics  Research and Applications</em> (ISBRA), LNCS 7875, pp. 64-75, 2013. (<a href="publications/Piyush_ISBRA_2013.pdf">PDF</a>) </p>
              </li>
              <li>
                <p>Wenting Liu, Kuiyu Chang, <strong>Jie Zheng</strong>, Jain Divya, Jung-Jae Kim, Jagath C. Rajapakse. <a href="http://link.springer.com/chapter/10.1007/978-3-642-38036-5_12">Gene Regulatory Networks from Gene Ontology</a>. <em>Proc. Bioinformatics  Research and Applications</em> (ISBRA), LNCS  7875, pp. 87-98, 2013. (<a href="publications/Liu_ISBRA_2013_GON.pdf">PDF</a>) </p>
              </li>
              <li>
                <p>Haifen Chen#, D.A.K. Maduranga# (joint first authors), Piyushkumar A. Mundra, <strong>Jie Zheng*</strong>. Integrating Epigenetic Prior in Dynamic Bayesian Network for Gene Regulatory Network Inference.<a href="http://www.ntu.edu.sg/home/epnsugan/index_files/SSCI2013/CIBCB2013.html"> Proc. IEEE Symposium on Computational Intelligence in Bioinformatics and Computational Biology</a> (CIBCB), pp. 76-82, Singapore, as part of <a href="http://www.ntu.edu.sg/home/epnsugan/index_files/SSCI2013/">IEEE Symposium Series on Computational Intelligence</a> (SSCI), <a href="http://www.ntu.edu.sg/home/epnsugan/index_files/SSCI2013/CIBCB2013.html"></a>2013. (<a href="publications/Chen_SSCI_2013_EpiGRN.pdf">PDF</a>)</p>
              </li>
              <li>
                <p>M. Wu, C-K Kwoh,  X-L Li, <strong>J.  Zheng*</strong>. NetPipe: A Network-based Pipeline for Discovery of Genes and Protein Complexes Regulating Meiotic Recombination Hotspots. <a href="http://www.cse.buffalo.edu/ACM-BCB2012/acceptedPapers.html">Proc. ACM Conference on Bioinformatics, Computational Biology and Biomedicine (ACM-BCB)</a>, Orlando, FL, USA, pp. 20-27, 2012. (<a href="publications/Wu_ACM-BCB_2012_NetPipe.pdf">PDF</a>)</p>
              </li>
              <li>
                <p>M. Wu, C-K Kwoh,  TM Przytycka, J. Li, <strong>J.  Zheng*</strong>. Integration of Genomic and Epigenomic Features  to Predict Meiotic Recombination Hotspots in Human and Mouse. <a href="http://www.cse.buffalo.edu/ACM-BCB2012/acceptedPapers.html">Proc. ACM Conference on Bioinformatics, Computational Biology and Biomedicine (ACM-BCB)</a>, Orlando, FL, USA, pp. 297-304, 2012. (<a href="publications/Wu_ACM-BCB_2012_SVM.pdf">PDF</a>) </p>
              </li>
              <li>
                <p>J. Mei, C-K Kwoh, P. Yang, X-L Li, <strong>J.  Zheng</strong>. <a href="http://dl.acm.org/citation.cfm?id=2350178">Globalized Bipartite Local Learning Model for Drug-Target Interaction Prediction</a>. <em>International Workshop on Data Mining in Bioinformatics (BIOKDD)</em>, Beijing, China, 2012. (<a href="publications/Mei-BioKDD-2012-submission.pdf">PDF</a>) </p>
              </li>
              <li>
                <p>P.  Yang, M. Wu, C-K Kwoh, PP Khil, RD Camerini-Otero, TM Przytycka, <strong>J. Zheng*</strong>. Predicting  DNA Sequence Motifs of Recombination Hotspots by Integrative Visualization and  Analysis. <em>Proc. International Symposium  on Integrative Bioinformatics</em>, pp. 52-58, 2012. (<a href="publications/Zheng_IB_2012_LDsplit.pdf">PDF</a>) </p>
              </li>
              <li>
                <p><strong class="STYLE7">J. Zheng</strong>, I. Chaturvedi, J. C. Rajapakse. <a href="http://www.springerlink.com/content/28m6g87v05w22278/">Integration of epigenetic data in Bayesian Network modeling of Gene Regulatory Network</a>. <em>The 6th IAPR International Conference on Pattern Recognition in Bioinformatics</em> (PRIB), LNCS 7036, pp. 87-96, 2011. (<a href="publications/Zheng_PRIB_2011_EpiGRN.pdf">PDF</a>) </p>
              </li>
              <li>
                <p>M. Wu, C-K Kwoh, TM Przytycka, J. Li, <strong>J. Zheng</strong>*. <a href="http://www.computer.org/portal/web/csdl/doi/10.1109/BIBM.2011.77">Prediction of trans-regulators of recombination hotspots in mouse genome</a>. <em>IEEE International Conference on Bioinformatics and Biomedicine</em> (BIBM), pp. 57-62, 2011. (<a href="publications/Zheng_BIBM_2011_mouseHotspot.pdf">PDF</a>) </p>
              </li>
              <li>
                <p>Z. Wang, H. Hu, <strong>J. Zheng</strong>*, B. Li* (co-corresponding authors). <a href="http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=6089810&url=http%3A%2F%2Fieeexplore.ieee.org%2Fiel5%2F6088020%2F6089804%2F06089810.pdf%3Farnumber%3D6089810">Gene expression and pathway analysis of quiescent CD8+ T  Cells from liver cancer, liver sinusoid and peripheral blood -- study  on toxicogenomics and prevention targeting</a>. <em>IEEE International Symposium on  BioInformatics and BioEngineering </em>(BIBE), pp. 72-77, 2011. (<a href="publications/Zheng_BIBE_2011_LiverCancer.pdf">PDF</a>)</p>
              </li>
              <li>
                <p><strong class="STYLE7">J. Zheng</strong>, IB Rogozin, EV Koonin, TM Przytycka. <a href="http://www.springerlink.com/content/l7710804w3861163/?MUD=MP">A rigorous analysis of the pattern of intron conservation supports the Coelomata clade of animals</a>. <em>RECOMB Comparative Genomics</em>, LNCS 4751, pp. 177-191, 2007.</p>
              </li>
              <li>
                <p><span class="STYLE7"><strong>J. Zheng</strong>, S. Lonardi. <a href="http://ieeexplore.ieee.org/xpl/freeabs_all.jsp?arnumber=1544455">Discovery  of repetitive patterns in DNA with accurate boundaries</a>. <em>Proc. of IEEE International Symposium on  BioInformatics and BioEngineering </em>(BIBE), pp. 105-112,  2005</span>.</p>
              </li>
              <li>                
                <p>X. Chen, <strong>J. Zheng</strong>, Z. Fu, P. Nan, Y. Zhong, S.  Lonardi and T. Jiang. Assignment  of orthologous genes via genome rearrangement. <em>Proc. of Asia-Pacific Bioinformatics  Conference </em>(APBC),  pp. 363-378, 2005</p>
              </li>
              <li>
                <p>A. Goldstein, P. Kolman, <strong>J. Zheng</strong> (authors in alphabetical order). <a href="http://www.springerlink.com/content/ml7l9u8le956tnp3/">Minimum  common string partition problem: hardness and approximation</a>. <em>Proc. of  International Symposium on Algorithms and Computation </em>(ISAAC),  pp. 484-495, LNCS 3341, 2004.</p>
              </li>
              <li>
                <p><strong class="STYLE6 STYLE7">J. Zheng</strong>, T. Close, T. Jiang,  S. Lonardi. <a href="http://www.springerlink.com/content/q967v058062571n1/">Efficient  Selection of Unique and Popular Oligos for Large EST Databases</a>. <em>Proc. of Symposium  on Combinatorial Pattern Matching </em>(CPM), pp. 384-401, LNCS 2676, 2003.</p>
              </li>
            </ol>
            <p class="STYLE5 STYLE23 STYLE6">&nbsp;</p>
            <p class="STYLE5 STYLE23 STYLE6">Book Chapters </p>
            <ol>
              <li>
                <p> Biaoru Li, Xiaomeng Zhang, <strong>Jie Zheng</strong>. <a href="http://link.springer.com/chapter/10.1007/978-3-319-15811-2_1">Single-Cell  Next-Generation Sequencing and Its Applications in Cancer Biology</a>. <em>Next Generation Sequencing in Cancer  Research</em>, Volume 2, pages 1 &ndash; 18, Springer International Publishing, 2015.</p>
              </li>
              <li>
                <p>Teresa M. Przytycka, <strong>Jie Zheng</strong>. <a href="http://www.els.net/WileyCDA/ElsArticle/refId-a0005267.html">Hidden Markov Models</a>. <em>Encyclopedia of Life Sciences,</em> 2011.</p>
              </li>
              <li>Yang  Huang*, <strong>Jie Zheng*</strong>, Teresa M. Przytycka (*co-first author).              Discovery  of regulatory mechanisms from gene expression variation by eQTL analysis. <em><a href="http://www.crcpress.com/product/isbn/9781420086843">Biological Data Mining</a></em>, Chapman and Hall/CRC Press, 2009 (<a href="publications\Zheng_BioDataMining_2009_eQTL.pdf">PDF</a>).</li>
            </ol>
            <p>&nbsp; </p>
          <!-- #EndEditable --></TD>
          <TD width="31%" valign="top"><form name="form1" method="post" action="">
            <!-- #BeginEditable "RightColumn" --><!-- #EndEditable -->
                                        </form>
          </TD>
        </TR>
      </TABLE>
    </TD>
    <TD width="9">&nbsp;</TD>
  </TR>
  <TR>
    <TD width="10"><IMG src="images/transparent.gif" alt="" width="10" height="30"></TD>
    <TD width="100%" valign="top">
      <HR>
      <P align="center" class="small"><A href="index.htm">Home</A><IMG src="images/bullet.gif" alt="" width="15" height="15"><A href="people.htm">People</A><IMG src="images/bullet.gif" alt="" width="15" height="15"><A href="teaching.htm"></A><A href="publications.htm">Publications</A><IMG src="images/bullet.gif" alt="" width="15" height="15"><A href="software.htm">Software</A><IMG src="images/bullet.gif" alt="" width="15" height="15"><A href="positions.htm">Positions</A><IMG src="images/bullet.gif" alt="boundary" width="15" height="15"><A href="contact.htm">Contact</A><BR>
        <IMG src="images/bullet.gif" alt="" width="15" height="15">&copy;
        <script language="JavaScript" type="text/javascript">
var d=new Date();
yr=d.getFullYear();
if (yr!=2000)
document.write(" "+yr);
        </script>
      ShanghaiTech University <IMG src="images/bullet.gif" alt="" width="15" height="15"></P>
      <div id="designed">
        <p align="center">&nbsp;</p>
      </div></TD>
    <TD width="9">&nbsp;</TD>
  </TR>
  <TR> 
    <TD width="10"><IMG src="images/bottomleft.gif" alt="" width="10" height="19"></TD>
    <TD background="images/bottommiddle.gif" width="100%">&nbsp;</TD>
    <TD width="9"><IMG src="images/bottomright.gif" alt="" width="9" height="19"></TD>
  </TR>
</TABLE>
</BODY>
<!-- #EndTemplate --></HTML>
